lib/fog/ovirt/requests/compute/v4/add_volume.rb

```diff
@@ -0,0 +1,53 @@
+module Fog
+  module Compute
+    class Ovirt
+      class V4
+        class Real
+          DISK_SIZE_TO_GB = 1024 * 1024 * 1024
+
+          def add_volume(id, options = {})
+            raise ArgumentError, "instance id is a required parameter" unless id
+
+            options = add_options_defaults(options)
+
+            disk_attachments_service = client.system_service.vms_service.vm_service(id).disk_attachments_service
+            disk = OvirtSDK4::DiskAttachment.new(options)
+            disk_attachments_service.add(disk)
+          end
+
+          # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+          def add_options_defaults(options)
+            search = options[:search] || format("datacenter=%<datacenter>s", :datacenter => datacenter)
+            options[:bootable] = "true" unless options[:bootable]
+            options[:interface] = OvirtSDK4::DiskInterface::VIRTIO unless options[:interface].present?
```

Comment:
``` ruby
options[:interface] ||= OvirtSDK4::DiskInterface::VIRTIO
```

The same for all other properties are setting default, if the property is not present.